private-templates-service/client/src/components/Common/SkipLink/styled.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import { COLORS } from "../../../globalStyles";
+
+export const SkipLinkContainer = styled.a`
+display: block;
+position: absolute;
+left: -999px;
+top: -999px;
```

Comment:
Is this how it is supposed to be implemented or did you just do this to make it hidden

Reply:
I just did it to move it off screen